src/react-pan-zoom.tsx

```diff
@@ -58,30 +60,32 @@ export default class ReactPanZoom extends React.PureComponent<IReactPanZoomProps
   public state = this.getInitialState();
 
   private onMouseDown = (e: React.MouseEvent<EventTarget>) => {
-    const { matrixData, dragData } = this.state;
-    const offsetX = matrixData[4];
-    const offsetY = matrixData[5];
-    const newDragData: IDragData = {
-      dx: offsetX,
-      dy: offsetY,
-      x: e.pageX,
-      y: e.pageY,
-    };
-    this.setState({
-      dragData: newDragData,
-      dragging: true,
-    });
-    if (this.panWrapper) {
-      this.panWrapper.style.cursor = "move";
+    if (this.props.enablePan) {
```

Comment:
@nsallis Can we invert this condition?

```
if (!this.props.enablePan) {
  return;
}
```

Reply:
@nsallis No worries. Just an integration test would be fine. You could follow the existing integration test under `integration_tests` folder and the new test that you will write would be the opposite of the `Should work on drag`.

Basically if you pass in the flag and perform the same test the matrix property on container shouldn't have changed. (https://github.com/ajainarayanan/react-pan-zoom/blob/master/integration_tests/react-pan-zoom.spec.js#L9)

Reply:
Thanks, that's very helpful and I learned something new. Let me know what you think. I'm new to typescript so if there's something I can do more semantically, please let me know.